packages/vaadin-grid/src/vaadin-grid-column-resizing-mixin.d.ts

```diff
@@ -6,8 +6,6 @@ interface ColumnResizingMixinConstructor {
 
 export { ColumnResizingMixinConstructor };
 
-interface ColumnResizingMixin {
-  ready(): void;
```

Comment:
Here I had a weird clash with the Polymer mixins, multiple of which also declare this method. Though the signature is the same, Typescript would return an error. I decided to remove this since it's not really a public API and the method is still present for all types which also extend from a PolymerElement.